src/hiwire.c

```diff
@@ -7,11 +7,11 @@ EM_JS(void, hiwire_setup, (), {
   {
     var objects = hiwire.objects;
     while (hiwire.counter in objects) {
-      hiwire.counter = (hiwire.counter + 1) % 0x8fffffff;
+      hiwire.counter = (hiwire.counter + 1) & 0x8fffffff;
```

Comment:
Isn't that only equivalent if the divisor is a power of 2, and we would need to run `& (divisor-1)` ?  It doesn't seem to be the case here, or am I missing something?

Reply:
You're correct.  Will fix.